index.js

```
@@ -1,3 +1,4 @@
+/*eslint strict: ["error", "function"]*/
```

Rather than ignoring the error, just add a `'use strict';` here.